components/mp2660/.eil.yml

```diff
@@ -0,0 +1,19 @@
+name: mp2660
+description: Driver for MP2660 5V USB, 500mA, I2C-Controlled Linear Charger with Power Path Management for Single-Cell Li-Ion Battery
+version: 1.0.0
+groups:
+  - misc
```

Comment:
the group should be `battery`, not `misc`

Reply:
Done. I searched for charger, not for battery :-)